examples/custom-vpc-network/main.tf

```diff
@@ -82,8 +82,6 @@ module "lacework_gcp_agentless_scanning_project_multi_region_usc1" {
     google = google.usc1
   }
 
-  project_filter_list = local.project_filter_list
```

Comment:
Is this removed to test that the module works without supplying the filter list?

Reply:
yes.